elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -134,7 +138,17 @@ public void registerTransformers(SubsystemTransformerRegistration registration)
     private static void from15(ChainedTransformationDescriptionBuilder chainedBuilder) {
         ResourceTransformationDescriptionBuilder builder = chainedBuilder.createBuilder(ELYTRON_15_0_0, ELYTRON_14_0_0);
         builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.JAAS_REALM));
-
+        
```

Comment:
Just FYI. Version 15 of the Elytron schema and model was already included in WildFly 26 so prior to completing this RFE, we'd need to bump the version again. Let's hold off on that bump for now though.